newIDE/app/src/ExportAndShare/Builds/BuildStepsProgress.js

```diff
@@ -34,6 +35,12 @@ const useStepLabelStyles = makeStyles(theme => ({
   },
 }));
 
+const usetStepConnectorStyles = makeStyles(theme => ({
+  root: {
+    flex: 0,
```

Comment:
forced to do this so that the stepper does not expand and take the full height of the modal